dxcherry/src/reviewers/orgstruct.ts

```diff
@@ -0,0 +1,52 @@
+import axios from 'axios';
+import { authentication } from 'vscode';
+import { logInfo } from '../info';
+import { USERS_DATA } from './config';
+import { msRefreshError } from '../info/errors/errors';
+
+async function getToken () : Promise<any> {
+  const session = await authentication.getSession('microsoft', ['openid'], { createIfNone: true });
+
+  logInfo(JSON.stringify(session.account));
+
+  try {
+    const response = await axios.post('https://resolve.devexpress.com/authorize', {
+      access_token: session.accessToken
+    }, {
+      headers: {
+        'Content-Type': 'application/json',
+        Accept: 'application/json'
```

Comment:
It would be better to wrap Accept into string too for consistance with Content-Type

Reply:
fixed

Reply:
Sorry, but our linter removes the quotes